faiss_vector_posting.go

```diff
@@ -266,7 +266,60 @@ func (vpl *VecPostingsIterator) BytesWritten() uint64 {
 	return 0
 }
 
-func (sb *SegmentBase) SimilarVectors(field string, qVector []float32, k int64, except *roaring.Bitmap) (segment.VecPostingsList, error) {
+func (sb *SegmentBase) GetVectorIndex(field string) (segment.VectorIndex, error) {
```

Comment:
Add a comment above here, that the caller is now responsible for closing the `vecIndex`.
Will add this in separate PR.